tools/src/main/scala/scala/scalanative/build/NativeLib.scala

```diff
@@ -1,27 +1,95 @@
 package scala.scalanative
 package build
 
+import java.io.File
 import java.nio.file.Path
+import java.util.regex._
 
 /** Original jar or dir path and generated dir path for native code */
 private[scalanative] case class NativeLib(src: Path, dest: Path)
 
+/** Utilities for dealing with native library code */
 private[scalanative] object NativeLib {
+  private val fs           = File.separator
```

Comment:
I read this as "file system", not as "file separator". Consider renaming it as `fileSep`.

Reply:
Ok sure thing.